app/admin/department.rb

```diff
@@ -19,4 +19,17 @@
       end
     end
   end
+
+  action_item :view, only: :index do
+    link_to 'Synchronise Departments', sync_admin_departments_path, method: :post
+  end
+
+  collection_action :sync, method: :post do
+    DeliveryOrganisationsImporter.new.import
+
+    mapping_file = File.read(Rails.root.join("app", "assets", "config", "department_mapping.csv"))
```

Comment:
I'm not sure this should live in `app/assets`, maybe `app/config` or `config`. `assets` is generally used for front-end resources; CSS, JS, images, fonts, etc.